color/src/commonMain/kotlin/ToColor.kt

```diff
@@ -0,0 +1,44 @@
+package com.juul.krayon.color
+
+private const val HEX = "[a-f0-9]"
+
+private val RGB = """($HEX)($HEX)($HEX)""".toRegex(RegexOption.IGNORE_CASE)
+private val RGBA = """($HEX)($HEX)($HEX)($HEX)""".toRegex(RegexOption.IGNORE_CASE)
+private val RRGGBB = """($HEX{2})($HEX{2})($HEX{2})""".toRegex(RegexOption.IGNORE_CASE)
+private val RRGGBBAA = """($HEX{2})($HEX{2})($HEX{2})($HEX{2})""".toRegex(RegexOption.IGNORE_CASE)
+
+public fun String.toColor(): Color =
+    keywordMap[this.toLowerCase()] ?: parseHexNotation(this)
```

Comment:
nit

Slight optimization could be to check for presence of `#` as first character (to remove hash lookup/miss) when you know you'll need to parse as hex notation.